public/app/features/dashboard-scene/edit-pane/MultiSelectedVizPanelsEditableElement.tsx

```diff
@@ -0,0 +1,51 @@
+import { SceneObject, VizPanel } from '@grafana/scenes';
+import { Button, Stack, Text } from '@grafana/ui';
+import { OptionsPaneCategoryDescriptor } from 'app/features/dashboard/components/PanelEditor/OptionsPaneCategoryDescriptor';
+
+import { MultiSelectedEditableDashboardElement } from '../scene/types';
+import { dashboardSceneGraph } from '../utils/dashboardSceneGraph';
+
+export class MultiSelectedVizPanelsEditableElement implements MultiSelectedEditableDashboardElement {
+  public isMultiSelectedEditableDashboardElement: true = true;
+
+  private items?: VizPanel[];
+
+  constructor(items: SceneObject[]) {
+    this.items = [];
+
+    for (const item of items) {
+      if (item instanceof VizPanel) {
+        this.items.push(item);
+      }
+    }
+  }
+
+  useEditPaneOptions(): OptionsPaneCategoryDescriptor[] {
+    return [];
+  }
+
+  public onDelete = () => {
+    for (const panel of this.items || []) {
+      const layout = dashboardSceneGraph.getLayoutManagerFor(panel);
+      layout.removePanel(panel);
+    }
+  };
+
+  public getTypeName(): string {
+    return 'Panels';
+  }
+
+  renderActions(): React.ReactNode {
+    return (
+      <>
+        <Stack direction={'column'}>
+          <Text>{`No. of panels selected: ${this.items?.length}`}</Text>
```

Comment:
Please use the `Trans` element here.

Reply:
hmm, I'm wondering: shouldn't betterer have warned me about this? It would help (*me*) in the future I think.